Dockerfile.scorecard

```diff
@@ -17,7 +17,13 @@ COPY test/scorecard ./test/scorecard
 # Build
 RUN GOOS=linux GOARCH=amd64 make scorecard/compile
 
-FROM kudobuilder/kuttl:v0.11.1
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
```

Comment:
I decided to use ubi8 as the base image. In OpenShift CI, this image gets replaced anyway, [also by ubi8](https://github.com/openshift/release/blob/0a274cf01ef647eb340e8c9ca7361a3de0ca75ac/ci-operator/config/integr8ly/integreatly-operator/integr8ly-integreatly-operator-master.yaml#L6-L9)